### DISCUSSION TITLE
bulk or surf and which cell_type

### DISCUSSION THREAD

**Discussion by McAlexDudu** [deepmodeling/dpgen#668] [Q&A] [answered]
Hi~ I had some base question about select init_bulk or init_surf and the cell_type. Since I work on battery material of LiGePS, I choose init_bulk, but I am not sure I set super_cell and cell_type right. Could any suggestion?

I choose from_poscar, and 

"super_cell":	[1, 1, 1],
"cell_type":    "sc",

<img width="260" alt="截屏2022-03-14 10 42 34" src="https://user-images.githubusercontent.com/16278257/158095806-dfc41fc7-d91a-4f4a-a7f3-398b581125e8.png">

**Comment by AnguseZhang** [comment, marked as answer]
Would the settings in the paper "Deep potential generation scheme and simulation protocol for the Li 10 GeP 2 S 12-type superionic conductors" (https://pubmed.ncbi.nlm.nih.gov/33685134/) help ?
